pilot/pkg/model/config_test.go

```diff
@@ -891,26 +891,20 @@ func TestServiceRoleBindings(t *testing.T) {
 
 func TestRbacConfig(t *testing.T) {
 	store := model.MakeIstioStore(memory.Make(model.IstioConfigTypes))
-	addRbacConfigToStore(model.RbacConfig.Type, "rbac-config", "istio-system", store, t)
+	addRbacConfigToStore(model.RbacConfig.Type, "rbac-config", "", store, t)
```

Comment:
use constant (DefaultRbacConfigName) instead of "rbac-config"

Reply:
Done